api/api.proto

```diff
@@ -2163,24 +2171,32 @@ message SdkCloudBackupStatus {
   google.protobuf.Timestamp completed_time = 6;
   // NodeID is the ID of the node where this Op is active
   string node_id = 7;
+  // SourceVolumeID is the the volume that is either being backed up to cloud
+  // or target volume to which cloudsnap is being restored
+  string src_volume_id = 8;
+  // ErrString indicates the failure cause for failed backup/restore
```

Comment:
Should this be `info`? We could use this for good status information also.

Reply:
you mean ErrInfo? this is expected to be non-empty only if the status says failed as the comment indicates.

Reply:
I mean just `info`. I would have suggested `status` but that is taken already. In a way I feel this is an object like in Kubernetes, where they have status, and users can read what is happening to the object when the check the status (which in kubernetes is a `[]string`).

I'm suggesting that this could be called `repeated string info`, where the server can continue adding information to the object and the caller can get a sequence of what happened (like a log).

I'm not 100% on this, just suggesting. 

If you do not think this is a good idea, then at least I would suggest changing the name to `error_details` instead of string.